pkg/account/authorization.go

```diff
@@ -141,18 +141,26 @@ func (c *authorizationComponentMW) UpdateAccount(ctx context.Context, account ap
 		return err
 	}
 
+	if !isEnabled(config.APISelfAccountEditingEnabled) {
```

Comment:
I don't see the difference of behavior between APISelfAccountEditingEnabled and APISelfMailEditingEnabled. Both needs to be enabled to be able to perform an edition. But it is not possible to allow to only edit the email. Should we enhance our mechanism to be able to provide finer grain of authorization for edition or remove the falg related to mail edition ?

Reply:
Removing APISelfMailEditingEnabled